src/sql/workbench/contrib/dashboard/browser/core/dashboardPage.component.ts

```diff
@@ -181,7 +181,7 @@ export abstract class DashboardPage extends AngularDisposable implements IConfig
 			let primary: IAction[] = [];
 			let secondary: IAction[] = [];
 			const menu = this.menuService.createMenu(MenuId.DashboardToolbar, this.contextKeyService);
-			let groups = menu.getActions({ arg: null, shouldForwardArgs: true });
+			let groups = menu.getActions({ arg: this.connectionManagementService.connectionInfo.connectionProfile.toIConnectionProfile(), shouldForwardArgs: true });
```

Comment:
Are there any existing tests around this? If yes, can it be enhanced to include this fix. If no, can it be added so that problems like these actually get caught beforehand.

Reply:
there currently aren't any tests for dashboardPage.component :( Since that'll be more involved than this small change, tests can be added in later PR.